tests/etl/test_transformers.py

```
@@ -0,0 +1,78 @@
+            "d": DummyColumn("object"),
+        }
+
+
+def build_transformer() -> transformers.KaggleTransformer:
+    t = transformers.KaggleTransformer()
+    transformers.SCHEMAS["test_table"] = DummySchema()
+    return t
+
+
+@given(st.lists(st.text(min_size=1), min_size=1, max_size=5, unique=True))
+def test_clean_column_names(cols):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_clean_column_names`

Add return type annotation: `None`

(ANN201)